tests/tests.js

```diff
@@ -81,23 +74,23 @@ define(['jquery', 'zimArchive', 'zimDirEntry', 'util', 'utf8'],
      * @param e Error
      */
     function errorHandlerAsyncTest(e) {
-        ok(false, "Error in async call", e);
-        start();
+        assert.ok(false, "Error in async call", e);
+        done();
     }
```

Comment:
Is this function still necessary?
It looks like you removed every reference to it, so I suppose it can be removed?